cli/actions/setup.js

```diff
@@ -86,6 +86,9 @@ module.exports = (config, flags, args) => {
       commands = uniq(commands.concat(tempScripts));
     }
 
+    // This is the default test script added by 'npm init'.
+    const npmInitDefaultTestScript = 'echo \"Error: no test specified\" && exit 1'; // eslint-disable-line no-useless-escape
```

Comment:
(or, alternately, it just ignores the escape character, in which case it's ok to omit the escapes from `npmInitDefaultTestScript` and remove the eslint-disable)